x-pack/plugins/lens/server/routes/field_stats.ts

```diff
@@ -206,15 +213,27 @@ export async function getNumberHistogram(
 
 export async function getStringSamples(
   aggSearchWithBody: (body: unknown) => unknown,
-  field: { name: string; type: string }
+  field: IFieldType
 ): Promise<FieldStatsResponse> {
+  const fieldRef: { field: string } | { script: { lang: string; source: string } } = field.scripted
```

Comment:
nitpick: there's just a bit of logic here so it's not a biggie, but maybe we could extract it to a function as it is used 3 times? I'm leaving the decision to you though

Reply:
Done